algorithm_catalog/vito/peakvalley/records/peakvalley.json

```diff
@@ -0,0 +1,174 @@
+{
+    "id": "peakvalley",
+    "type": "Feature",
+    "conformsTo": [
+        "http://www.opengis.net/spec/ogcapi-records-1/1.0/req/record-core",
+        "https://apex.esa.int/core/openeo-udp"
+    ],
+    "geometry": null,
+    "properties": {
+        "created": "2025-01-09T00:00:00Z",
+        "updated": "2025-01-29T00:00:00Z",
+        "type": "service",
+        "title": "Detect peaks and valleys in a time series",
+        "description": "# Peak Valley Detection\n\n## Overview\n\nThe `peakvalley` process provides automated detection of peaks and valleys in time-series data by analysing amplitude changes and slope patterns. It identifies significant drops, recoveries, and inflexion points to classify each time step as a peak, a valley, or a neutral state. \nThis process is particularly useful for applications such as vegetation phenology monitoring, hydrological studies, and climate data analysis.\n\n## Parameters\n\nThe `mogpr_s2` service requires the following parameters:\n\n\n| Name            | Description                                                    | Type    | Default |\n| --------------- | -------------------------------------------------------------- | ------- | ------- |\n| spatial_extent  | Polygon representing the AOI on which to apply the data fusion | GeoJSON |         |\n| temporal_extent | Date range for which to apply the data fusion                  | Array   |         |\n| drop_threshold  | Threshold to drop low confidence predictions                   | Float   | 0.15    |\n| recovery_ratio  | Ratio to recover from drops in the data                        | Float   | 1.0     |\n| slope_threshold  | Threshold to identify steep slopes in the data                | Float   | 0.007   |\n",
```

Comment:
This description should be a short description as it is shown in the sidebar. Can you please summarise this text into a shorter description as you did for the mogpr_s1s2?